crates/node/builder/src/rpc.rs

```diff
@@ -147,6 +148,7 @@ impl<Node: FullNodeComponents> ExtendRpcModules<Node> for () {
 
 /// Helper wrapper type to encapsulate the [`RethModuleRegistry`] over components trait.
 #[derive(Debug)]
+#[allow(clippy::type_complexity)]
```

Comment:
Maybe we can create type aliases to avoid this allow no?

Reply:
ref https://github.com/paradigmxyz/reth/pull/9041#discussion_r1654457204